aiida_quantumespresso/calculations/__init__.py

```diff
@@ -24,6 +26,44 @@ class BasePwCpInputGenerator(CalcJob):
     _DATAFILE_XML_POST_6_2 = 'data-file-schema.xml'
     _ENVIRON_INPUT_FILE_NAME = 'environ.in'
 
+    _ENABLED_PARALLELIZATION_FLAGS = []
+    # Tuples (flag_name, default, help) of parallelization flags possible
+    # in QE codes. The flags that are actually implemented in a given
+    # code should be specified in the '_ENABLED_PARALLELIZATION_FLAGS'
+    # list of each calculation subclass.
+    _ALLOWED_PARALLELIZATION_FLAGS = (  # yapf: disable
```

Comment:
Two things:
1. Find the prefix "allowed" a bit weird. Why not just `_PARALLELIZATION_FLAGS`?
2. Why not make this a dictionary with the flag name as the key? I think you only use it in line 125 and there you transform it in a dict. Might as well define it as one then and save those lines

Reply:
Yeah good point, will change that.

Reply:
Done!